Libraries/Components/TextInput/InputAccessoryView.ios.js

```diff
@@ -88,10 +87,6 @@ type Props = $ReadOnly<{|
 
 class InputAccessoryView extends React.Component<Props> {
   render(): React.Node {
-    if (Platform.OS !== 'ios') {
-      console.warn('<InputAccessoryView> is only supported on iOS.');
-    }
```

Comment:
From an internal discussion:

Since InputAccessoryView is kind of like a "portal" (i.e. components render output is rendered inside a special native container), I think it would be better to make this return `null`.

Also, we wouldn't require call sites to use Flow suppressions or hacks.
If we wanted to do this a bit more efficiently (with dead code elimination), we could do something like:

```
function InputAccessoryView(props: Props): React.Node {
  if (Platform.OS === 'ios') {
    // Lines 95 - 106.
  } else {
    console.warn('…');
    return null;
  }
}
module.exports = InputAccessoryView;
```

cc @yungsters